### PR TITLE
built the router and main components

### PR DESCRIPTION
**Pull request by raulmogos**

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { BrowserRouter, Route } from 'react-router-dom';
+
+import Home from './Home';
+import Contacts from './Contacts';
```
Comment (raresfelecan) [inline src/components/App.js]: Maybe rename 'Contacts' to something like ContactsPage. In the future you'll have a 'ContactsList' and a 'Contact' component. 'Contacts' naming might be misleading in this case.

File: src/components/Header.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
```
Comment (orian-georgian) [inline src/components/Header.js]: Remove extra blank line

File: src/components/App.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+import Header from './Header';
+import Menu from './Menu';
+import { NAV } from '../constants/constants';
+
```
Comment (orian-georgian) [inline src/components/App.js]: One extra blank line is enough between class component and imports. Do that for all the components.

File: src/components/Home.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+
+function Home() {
+  return (
+    <div>
+      <h1 className="ui center aligned header">
+        welcome to the jungle
```
Comment (orian-georgian) [inline src/components/Home.js]: In the jungle, the mighty jungle, the lion sleeps tonight.

File: src/components/Menu.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
```
Comment (orian-georgian) [inline src/components/Menu.js]: Remove extra blank line.

File: src/index.js [hunk 6]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
```
Comment (orian-georgian) [inline src/index.js]: Remove extra blank line.

**Pull request merged**